_posts/2020-08-11-backports-application.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: From zero to hero on backporting pull requests
+date: 2020-08-11
+tags: user-story
+author: gastaldi
+---
+= From zero to hero on backporting pull requests
+
+The Quarkus project moves fast and when we prepare bugfix releases, we usually have several dozens of pull requests to backport. Using the GitHub UI to do that is feasible but cumbersome.
```

Comment:
These 2 paragraphs don't follow well.

And we say at least 3 times that we have many pull requests to process, which is why it's tedious. We should probably pull that here into the 1st paragraph, and then adjust the headings so that the text flows nicely. Something like:

```
= From zero to hero on backporting pull requests

The Quarkus project moves fast and when we prepare bugfix releases, we usually have several dozens of pull requests to backport.
Using the GitHub UI to do that is feasible but cumbersome.
So we decided to automate this work, and built an application.
Of course, with Quarkus!

Before we dig into it, let me give you a quick explanation of our release process.

== Release process

...

[remove the last paragraph "Considering the amount of backports ..."]

== How do we automate that

...

== Our experience with Quarkus

Yes, this would be instead of "Extensions", and all other headings (Live Coding, MP Config, Qute, Next steps) would be `===` under "Our experience"

...

== Thanks

...
```

That should work better I think.